app/views/shared/_nav.html.erb

```diff
@@ -7,22 +7,22 @@
     <ul class='navbar-nav ml-lg-auto d-flex flex-row'>
       <% if session[:user_id] %>
         <li class='nav-item px-lg-2'>
-          <%= link_to 'Logout', '/sessions', method: :post, class: 'btn btn-primary', style: 'margin-right: 1rem; background-color: #007bff; color: #fff; padding: 10px 20px; border-radius: 5px;', class: ('active' if current_page?('/sessions')) %>
+          <%= button_to 'Logout', '/sessions', method: :delete, class: 'btn btn-primary', style: 'margin-right: 1rem; background-color: #007bff; color: #fff; padding: 10px 20px; border-radius: 5px;' %>
```

Comment:
I will reorder these, and try to change to link_to, or modify buttons so they are hidden.